README.md

```diff
@@ -41,9 +41,9 @@ The docker configurations shown here are example use cases where the URxx manipu
     ```
     > **Note** 💡
     >
-    > You need to specify the volume path to your reference package in [compose](https://github.com/husarion/ur-docker/blob/9bf6dfb80d2dac7b4e5c7fb9835e1a80c7d3afe6/demo/compose.moveit-menager.yaml#L17) file. 
+    > You need to specify the volume relative path to your reference package in [compose](demo/compose.moveit-menager.yaml#L10) file. 
```

Comment:
Instead of a note here, we could add a step to add a volume to the compose file before the step with running docker compose. Looking at the file this is a required step and a note can be easily missed.